docs/release_notes.rst

```diff
@@ -59,6 +59,13 @@ SEC 10-K
 Expanded Data Coverage
 ^^^^^^^^^^^^^^^^^^^^^^
 
+FERC Form 1
+~~~~~~~~~~~
+* Integrated FERC Form 1 data from 2024 into the main PUDL SQLite DB. See issue
+  :issue:`4207` and PR :pr:`4215`. FERC Form 1 responses are submitted on a rolling
+  basis with no definitive completion. This update includes most ~90% of the expected
+  utility responses for 2024.
+
```

Comment:
this is my attempt at: hey ferc doesn't have actual "final" releases this is probably most of it

Reply:
ah yes! is this better? i don't know if the last sentence makes it more confusing.